src/Quadrature/gaussquad_tri_table.jl

```diff
@@ -72,6 +72,584 @@ function _get_gauss_tridata(n::Int)
     0.72849239295540 0.26311282963464 0.02723031417443 / 2.0
     0.26311282963464 0.00839477740996 0.02723031417443 / 2.0
     0.00839477740996 0.72849239295540 0.02723031417443 / 2.0];
+    elseif n == 9
+    xw=[0.3333333333333333 0.3333333333333333 0.0971357962827987 / 2.0
```

Comment:
The first line in this file has a reference to where the numbers were taken from. I guess these are not taken from the same place? In that case, they should also have a reference how they got computed or where they were taken from.

Reply:
Thank you @KristofferC. I add a reference at the begining of the file. Not sure this is the right way to do it.

@fredrikekre you are perfectly right. This is very suspicious and you can observe the same surprising data points in the original paper of D.A. Dunavant.
I am not an expert of the wild world of quadrature formula that I just discovered.
At this point I do not know exactly what to do. Empirically test the quadrature orders obtained with these data points/weights (when the quadrature points are admissible)? Focus on other quadrature approaches which are used in well established libraries? 

It would be great to have high precision quadrature routines available (Float64, BigFloat..).

Reply:
A list of options with references [quadrature formula](https://www.math.unipd.it/~alvise/SETS_CUBATURE_TRIANGLE/rules_triangle.html)